regions_www/m/pct_shiny_funs.R

```diff
@@ -43,18 +43,10 @@ round_dp <- function(expression, small_threshold = 0.05, large_threshold = 0.5){
   expression
 }
 
-round_percent <- function(data, expression){
- value <- round(100 * expression)
-  for(i in 1:length(data)) {
-    if(!is.na(expression[i]) && !is.nan(expression[i]) && expression[i]>0 && expression[i]<0.005) {
-      value[i] <- round(100 * expression[i], 1)
-    }
-    if (is.nan(value[i]) || is.na(value[i])) { value[i] <- "-" }
-  }
- value
+round_percent <- function(expression){
+  round_dp(expression * 100, small_threshold = 0.0, large_threshold = 0.005)
```

Comment:
I think the large threshold value needs to be 0.5 not 0.005, since I think the *100 comes first and then the threshold is applied? Unless I make this change, the percentage shown below appears as 0% not 0.5%

![untitled](https://user-images.githubusercontent.com/11818225/35634912-c7a9cd86-06a4-11e8-85a4-5c8ff744b5d0.png)